selftests/functional/test_basic.py

```diff
@@ -391,6 +404,26 @@ def test_fail_on_exception(self):
                                                                 result))
         self.assertIn('"status": "FAIL"', result.stdout)
 
+    def test_exception_not_in_path(self):
+        os.chdir(basedir)
+        os.mkdir(os.path.join(self.tmpdir, "shared_lib"))
+        mylib = script.Script(os.path.join(self.tmpdir, "shared_lib",
+                                           "mylib.py"),
+                              "from avocado import TestCancel\n\n"
+                              "class CancelExc(TestCancel):\n"
```

Comment:
Well this is just 2 lines, I don't think it makes sense to add another 2 lines in front of it and introduce yet another global variable just to use it in one place...